scripts/update_config.js

```diff
@@ -79,6 +79,9 @@ oldConfig.peers.list = oldConfig.peers.list.map(p => {
 	return p;
 });
 
+// Move SSL to API only
+oldConfig.api.ssl = extend(true, {}, oldConfig.ssl);
```

Comment:
Should this be `newConfig.api.ssl = ...`? Also, shouldn't `oldConfig.ssl` be deleted?
It looks a bit different than how it was implemented in https://github.com/LiskHQ/lisk/pull/2197/files#diff-02f64463a1deb2818037febeeb0fa51d